Sources/SwiftSyntax/SwiftSyntaxCompatibility.swift

```diff
@@ -315,3 +315,228 @@ public extension EditorPlaceholderDeclSyntax {
     )
   }
 }
+
+public extension AvailabilityConditionSyntax {
```

Comment:
Could you add the types in alphabetical order? And maybe add a comment at the end of the file similar to the following? When I started this file, I didn’t think it would grow as much that quickly but now that it does, I think some structure would be good.


https://github.com/apple/swift-syntax/blob/a90e1be0bb5171e6da973946e712ade67526c207/Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift#L1783-L1786


Also, just an idle thought: If we continue to rename properties like this, it might be worth investing in Code-Generation of the file.

Reply:
I agree with you. I'll add comments and organize the code alphabetically.